src/test/java/org/broadinstitute/hellbender/engine/GATKToolUnitTest.java

```diff
@@ -523,6 +527,28 @@ private File setupVCFWriter(
         return outputFile;
     }
 
+    @Test
+    public void testGetDefaultToolVCFHeaderLines() throws IOException {
+        final TestGATKToolWithFeatures tool = new TestGATKToolWithFeatures();
+        final File vcfFile = new File(publicTestDir + "org/broadinstitute/hellbender/engine/feature_data_source_test_with_bigHeader.vcf");
+        final String[] args = {"--mask", vcfFile.getCanonicalPath(), "--" + StandardArgumentDefinitions.ADD_OUTPUT_VCF_COMMANDLINE, "true"};
+        tool.instanceMain(args);
+
+        Set<VCFHeaderLine> stdHeaderLines = tool.getDefaultToolVCFHeaderLines();
+        VCFHeader hdr = new VCFHeader(stdHeaderLines);
+
+        VCFHeaderLine sourceLine = hdr.getOtherHeaderLine("source");
+        Assert.assertEquals(sourceLine.getValue(), tool.getClass().getSimpleName());
+
+        VCFIDHeaderLine commandLine = (VCFIDHeaderLine) hdr.getOtherHeaderLine("GATKCommandLine");
+        Assert.assertEquals(commandLine.getID(), tool.getClass().getSimpleName());
+
+        String commandLineString = commandLine.toString();
+        Assert.assertTrue(commandLineString.contains("CommandLine="));
```

Comment:
we have BaseTest.assertContains(String,String) which does this but with a nicer error message.